sdjournal/journal_test.go

```diff
@@ -68,3 +68,27 @@ func TestJournalFollow(t *testing.T) {
 		t.Fatalf("Error during follow: %s", err)
 	}
 }
+
+func TestJournalGetUsage(t *testing.T) {
+	j, err := NewJournal()
+
+	if err != nil {
+		t.Fatalf("Error opening journal: %s", err)
+	}
+
+	if j == nil {
+		t.Fatal("Got a nil journal")
+	}
+
+	defer j.Close()
+
+	s, err := j.GetUsage()
+
+	if err != nil {
+		t.Fatalf("Error getting journal size: %s", err)
+	}
+
+	if s == 0 {
+		t.Fatalf("Journal size is 0.")
```

Comment:
It seems odd to fatal on 0 when the sd_journal_get_usage() manual says nothing about 0 size being invalid